deploy/kubernetes/auditbeat-kubernetes.yaml

```diff
@@ -243,6 +257,24 @@ rules:
     - replicasets
   verbs: ["get", "list", "watch"]
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: auditbeat
+  namespace: kube-system
+  labels:
+    k8s-app: auditbeat
+rules:
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs: ["get", "create", "update"]
+  - apiGroups: [""]
+    resources:
+      - configmaps
+    verbs: ["get"]
```

Comment:
Nit. I am wondering if this role may worth a comment now, or these two rules should be in different `Roles`. The thing is that the leases rule need to apply to the same namespace where the beat is deployed, but this configmap rule needs to apply specifically to the `kube-system` namespace, where `kubeadm-config` is deployed.

Reply:
Good idea!